pkg/tfvars/vsphere/vsphere.go

```diff
@@ -0,0 +1,55 @@
+package vsphere
+
+import (
+	"encoding/json"
+
+	vsphereprovider "sigs.k8s.io/cluster-api-provider-vsphere/api/v1alpha3"
+	"sigs.k8s.io/cluster-api-provider-vsphere/api/v1alpha3/cloudprovider"
```

Comment:
there is a good chance this will not available in downstream openshift because we have other objects that provide this information.